websub-ballerina/commons.bal

```diff
@@ -93,7 +93,7 @@ public type SubscriptionVerification record {
 public type ContentDistributionMessage record {
     map<string|string[]>? headers = ();
     string? contentType = ();
-    map<string[]>|json|xml|string|byte[] content;
+    map<string>|json|xml|string|byte[] content;
```

Comment:
In the case application/x-www-form-urlencoded, can't we have the same key repeated ?

foo=foovalue&bar=barvalue&foo=foovalue2

Reply:
* we can have the payload as this. `http:Request` to retrieve form-urlencoded content we can use `request.getFormParams()` method. But it only support `map<string>` return type. Documentation could be found [here](https://ballerina.io/learn/api-docs/ballerina/#/ballerina/http/1.1.0-alpha4/http/classes/Request#getFormParams).  Underlying implementation is as [follows](https://github.com/ballerina-platform/module-ballerina-http/blob/e47774f8610453cb1e16c1983287211e844a01ef/http-ballerina/http_request.bal#L352).